src/style.css

```
@@ -0,0 +1,80 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+}
+
+.flex {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.flex__item {
+  display: flex;
+  height: 300px;
+  justify-content: center;
+  align-items: center;
+  flex-basis: 300px;
+  flex-grow: 1;
+  flex-shrink: 0;
+  color: #fff;
+}
+
+.flex__item--1 {
+  background-color: #f00;
+}
+
+.flex__item--2 {
+  background-color: #c00;
+}
+
+.flex__item--3 {
+  background-color: #900;
+}
+
+.flex__item--4 {
+  background-color: #600;
+}
+
+.flex__item--5 {
+  background-color: #300;
+}
+
+.flex__item--6 {
+  background-color: #000;
+}
+
+@media (min-width: 600px) {
+  .flex {
+    flex-direction: row;
+  }
+
+  .flex__item--1,
+  .flex__item--2,
+  .flex__item--4 {
+    order: -1;
```

```suggestion
  .flex__item--1,
  .flex__item--2,
  .flex__item--4 {
    order: -1;
```
Better not to give couple of blocks the same order (even though it's negative). Better to specify order for each specific element